x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/index.tsx

```diff
@@ -243,6 +251,18 @@ const UnifiedTimelineComponent: React.FC<Props> = ({
     sort: sortingColumns,
   });
 
+  const onSetColumnsTimeline = useCallback(
+    (nextColumns: string[]) => {
+      const shouldUnifiedTableKeepColumnsUnchanged = true;
+      // to support the legacy table, unified table has the ability to automatically
```

Comment:
Thanks for the note and giving the boolean a good name 😄 .  This still won't be able to be removed even when the new table is fully adopted then right?

Reply:
They have a comment that this can be removed when their legacy table is retired.

I will ask them about it.